src/components/transactions-list/index.jsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import cn from 'classnames';
 import { useTranslation } from 'react-i18next';
 
-import { ProfileTransactionsItem } from './item';
-import { Text } from '../../ui/Text';
-import { Button } from '../../ui/Button';
+import { TransactionsListItem } from './item';
+import { Text } from '../ui/Text';
+import { Button } from '../ui/Button';
 
 import style from './styles.module.scss';
 
-export const ProfileTransactions = ({ count, className }) => {
+export const TransactionsList = ({ primary, count, className }) => {
```

Comment:
Grammatically it should be `TransactionList`

Reply:
Done